easybuild/easyconfigs/n/NEST/NEST-2.12.0-foss-2015b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'NEST'
+version = '2.12.0'
+
+homepage = 'http://www.nest-simulator.org/'
+description = """NEST is a simulator for spiking neural network models
+ that focuses on the dynamics, size and structure of neural systems rather
+ than on the exact morphology of individual neurons."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['nest-%(version)s.tar.gz']
+source_urls = ['https://github.com/nest/nest-simulator/releases/download/v%(version)s/']
+
+dependencies = [
+    ('Python', '2.7.9'),
+    ('libreadline', '6.3'),
+    ('GSL', '2.1'),
+    ('libtool', '2.4.6', '-GNU-4.9.3-2.25', True),
```

Comment:
@jdonners There should be no need to hardcode the toolchain anymore, since EasyBuild v3 subtoolchains are considered automatically.

Also, isn't `libtool` a build dependency only?

And also, rather than using `libtool` as (build) dependency, we usually use the `Autotools` bundle as a build dep instead:

```python
builddependencies = [('Autotools', '20150215')]